frontend/index.html

```diff
@@ -5,73 +5,84 @@
   <meta charset="utf-8">
   <title>marcel-plugin-etherpad-board</title>
   <link rel="stylesheet" type="text/css" href="styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700" rel="stylesheet">
   <script src="./marcel.js"></script>
 </head>
 
 <body>
 
   <div id="container" class="container">
-    <span class="time">
+    <div class="date">
+      <span id="date"></span>
+    </div>
+    <div class="period">
+      <span id="period"></span>
+    </div>
+    <div class="time">
       <span id="hour"></span>
-      <span class="blink_me">:</span>
+      <span>:</span>
       <span id="minutes"></span>
-    </span>
-    <span class="date">
-      <span id="day"></span>
-      <span id="month"></span>
-      <span id="year"></span>
-    </span>
+    </div>
   </div>
 
   <script>
     const leftPad = number => ('0' + number).slice(-2)
```

Comment:
I know this is not part of your PR, but we could use the [standardized padStart](https://mdn.io/String/padStart)